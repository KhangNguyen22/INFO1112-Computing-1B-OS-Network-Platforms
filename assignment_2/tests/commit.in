connect bitbash.xyz 9050
login  oei-svr moasoiac
observe
commit
quit