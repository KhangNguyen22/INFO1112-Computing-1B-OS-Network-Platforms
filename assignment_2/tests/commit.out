> Connected, please log in
> Logged In!
> 
|  |  | 2|  |  | -|  |
|  |  |  | -|  |  |  |
|  | 7|  |RR|  |  |  |
|  |  |  |  | -|  |  |
|  | -|  |  |  |  |  |
> > 